doc_src/index.hdr.in

```diff
@@ -1129,7 +1131,7 @@ History searches can be aborted by pressing the escape key.
 
 Prefixing the commandline with a space will prevent the entire line from being stored in the history.
 
-The history is stored in the file `~/.local/share/fish/fish_history` (or `$XDG_DATA_HOME/fish/fish_history` if that variable is set).
+The command history is stored in the file `~/.local/share/fish/fish_history` (or `$XDG_DATA_HOME/fish/fish_history` if that variable is set) by default. However, you can set the `FISH_HISTFILE` environment variable to change the file name; both before starting the shell and while the shell is running.
```

Comment:
Not sure how to rephrase this but it incorrectly implies that FISH_HISTORY controls the absolute path name.